pkgs/development/python-modules/hikari/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pytest-runner
+, aiohttp
+, attrs
+, multidict
+, colorlog
+, pynacl
+, git
+, pytest
+, pytest-cov
+, pytest-randomly
+, pytest-asyncio
+, mock
+  # Opts
+, enableRestServer ? false
```

Comment:
```suggestion
```
We should not add parameters like this.
It would be better to use `optional-dependencies`.

Reply:
Fixed.